include/nethack_client.h

```diff
@@ -82,8 +82,8 @@ extern EXPORT nh_bool nhnet_view_replay_step(struct nh_replay_info *info,
                                              int count);
 extern EXPORT void nhnet_view_replay_finish(void);
 extern EXPORT struct nh_cmd_desc *nhnet_get_commands(int *count);
-extern EXPORT struct nh_cmd_desc *nhnet_get_object_commands(
-    int *count, char invlet);
+extern EXPORT struct nh_cmd_desc *nhnet_get_object_commands(int *count,
+                                                            char invlet);
```

Comment:
There's only whitespace changes here, right? If so, we should probably not apply it as to keep the diversion from upstream as small as possible.